packages/common/src/store/cache/reducer.ts

```diff
@@ -100,6 +99,12 @@ export const mergeCustomizer = (objValue: any, srcValue: any, key: string) => {
     return srcValue || objValue
   }
 
+  // Not every user request provides playlist_library,
+  // so always prefer it's existence, starting with latest
+  if (key === 'playlist_library') {
+    return objValue ?? srcValue
```

Comment:
nit we do || above. maybe we should be consistent?

Reply:
yeee sure thing